target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
I managed to build the BOSS header, but the kernel doesn't boot yet.

```
define Build/rs353-cev
        rm -f $(KDIR)/boss.bin.gz
        # Add CEV header (magic). The name "boss.bin" have to be in the gzip header.
        echo -ne "\x10\x00\x00\x02\x00\x00\x00\x00\x43\x45\x56\x00\x00\x00\x00\x00" > $(KDIR)/boss.bin
        # Append data
        cat $@ >> $(KDIR)/boss.bin
        # gzip the file
        gzip $(KDIR)/boss.bin
        # Align with flash sector size
        dd if=$(KDIR)/boss.bin.gz of=$@ bs=8 conv=sync
endef

define Build/rs353-boss-header
        cp $@ ./boss.tmp
        # Boss magic for Bintec RS353
        echo -n "TELDAT ClosedEyeVisual" > $(KDIR)/boss.img
        # Append the update version and separator
        echo -ne "\x00\x00\x2A\x02\x00\x00\x00\x00\x00\x00" >> $(KDIR)/boss.img
        # Append the file size
        printf "%08x\n" $(shell stat -c%s boss.tmp ) | xxd -r -p >> $(KDIR)/boss.img
        # Append the image type
        echo -ne "\x00\xFA\x05\x24\x00\x00\x00\x00" >> $(KDIR)/boss.img
        # Append the crc32
        crc32 boss.tmp | xxd -r -p >> $(KDIR)/boss.img
        # Separator
        echo -ne "\x00\x00\x00\x00" >> $(KDIR)/boss.img
        # Append the file content
        cat boss.tmp >> $(KDIR)/boss.img
        cat $(KDIR)/boss.img > $@
endef

define Device/RS353
  DEVICE_PROFILE := RS353
  DEVICE_TITLE := Bintec RS353
  KERNEL := kernel-bin | append-dtb | lzma | lzma-loader | rs353-cev
  IMAGE/boss-image.cev = append-kernel | append-rootfs | pad-rootfs | rs353-boss-header
  IMAGES := sysupgrade.bin boss-image.cev
  IMAGE_SIZE := 31232k
  DEVICE_PACKAGES := kmod-usb-dwc2 kmod-ath9k wpad-mini kmod-i2c-gpio kmod-swconfig wpad-mini kmod-ledtrig-usbdev
endef
TARGET_DEVICES += RS353
```
But...

```
### RS353aw (Hardware-Rev. 1.0, Firmware-Rev. 1.0) ###

CPU   Check ... passed (MIPS 34KEc [MT] @ 500/250.0 MHz)
SDRAM Check ................................................................................................................................. passed (128 MByte)
FLASH Check  passed (32 MByte)

### Selftest passed ###


Press <sp> for boot monitor or any other key to boot system

Booting Image from Flash ROM
Checking image ... OK
Writing image to RAM (Release 42.2) ....................................................... OK (1784183 bytes)
Booting BOSS...
boss image started at 0x2d6a0
Disabling WDT
EBU config


OpenWrt kernel loader for MIPS based SoC
Copyright (C) 2011 Gabor Juhos <juhosg@openwrt.org>
Decompressing kernel... 
```
It stops here.